reference/forms/types/collection.rst

```diff
@@ -363,6 +364,29 @@ rendering your form, having the entire "form row" may be easier for you.
 For details on how to actually use this option, see the above example as
 well as :ref:`cookbook-form-collections-new-prototype`.
 
+prototype_data
+~~~~~~~~~~~~~~
+
+**type**: ``mixed`` **default**: ``null``
+
+Allows you to define a specific data for the prototype. Each new row added will
+initially contain the data set by this option.
+By default, the same data as defined with ``entry_options`` for other entries
```

Comment:
There's no need for the newline (to begin a new paragraph you would need a blank line).